### DISCUSSION TITLE
is it possible extend SPDLOG_LEVEL_DEBUG to multiple levels?

### DISCUSSION THREAD

**Discussion by zhouqingqing** [gabime/spdlog#3108] [Q&A]
This is useful when we need to have different verbosity/levels of debug message. For example, PostgreSQL has 5 debug levels (DEBUG5, DEBUG4, ..., DEBUG1). At DEBUG5, it flushes screen with lots of tedious information, while at DEBUG1, it outputs high level information only. 

PostgreSQL ref: https://www.postgresql.org/docs/current/runtime-config-logging.html#RUNTIME-CONFIG-LOGGING-WHEN

**Comment by tt4g** [comment]
It is not possible to customize log levels in spdlog.

**Reply by zhouqingqing** [reply]
> It is not possible to customize log levels in spdlog.

Thank you for the answer. If customization is not possible,  do you think add more members to "enum  level_enum" is an acceptable patch?

**Reply by tt4g** [reply]
It is possible; it is OSS and there is no reason to stop it.
